go/services/multiadmin/init.go

```diff
@@ -98,11 +102,25 @@ func (ma *MultiAdmin) Init() error {
 	)
 
 	ma.senv.OnRun(func() {
-		// Register multiadmin gRPC service with servenv's GRPCServer
+		// Register multiadmin gRPC and HTTP API services if enabled in service map
 		if ma.grpcServer.CheckServiceMap(constants.ServiceMultiadmin, ma.senv) {
 			ma.adminServer = NewMultiAdminServer(ma.ts, logger)
 			ma.adminServer.RegisterWithGRPCServer(ma.grpcServer.Server)
-			logger.Info("MultiAdmin gRPC service registered with servenv")
+
+			// Set up grpc-gateway for REST API
+			gwmux := runtime.NewServeMux(
+				runtime.WithMarshalerOption(runtime.MIMEWildcard, &runtime.JSONPb{
+					MarshalOptions:   protojson.MarshalOptions{EmitUnpopulated: true, UseProtoNames: true},
+					UnmarshalOptions: protojson.UnmarshalOptions{DiscardUnknown: true},
+				}),
+			)
+			//nolint:gocritic // grpc-gateway handler registration runs at startup before request context is available
+			if err := multiadminpb.RegisterMultiAdminServiceHandlerServer(context.Background(), gwmux, ma.adminServer); err != nil {
```

Comment:
FYI, `cobra.Command` creates a context so one exists, it just may not be propagated here through function calls